packages/flutter/lib/src/material/carousel.dart

```diff
@@ -15,31 +16,60 @@ import 'theme.dart';
 
 /// A Material Design carousel widget.
 ///
-/// The [CarouselView] present a scrollable list of items, each of which can dynamically
+/// The [CarouselView] presents a scrollable list of items, each of which can dynamically
 /// change size based on the chosen layout.
 ///
+/// Material Design 3 introduces 4 carousel layouts:
+///  * Multi-browse: This layout shows at least one large, medium, and small
+/// carousel item at a time.
+///  * Uncontained (default): This layout show items that scroll to the edge of
+/// the container.
+///  * Hero: This layout shows at least one large and one small item at a time.
+///  * Full-screen: This layout shows one edge-to-edge large item at a time and
+/// scrolls vertically.
+///
 /// This widget supports uncontained carousel layout. It shows items that scroll
```

Comment:
Should this say: The default constructor implements the uncontained layout model.